pkg/services/ngalert/models/alert_rule.go

```diff
@@ -105,6 +105,14 @@ func (alertRule *AlertRule) PreSave(timeNow func() time.Time) error {
 	return nil
 }
 
+func (alertRule *AlertRule) GetResourceTypeIdentifier() string {
```

Comment:
We only add support for `AlertRule` for now. In the future, more objects will use this, but this one is chosen for tests.